descriptors/ModuleDescriptor-template.json

```diff
@@ -281,6 +281,17 @@
         }
       ]
     },
+    {
+      "id": "metadata-provider-record-processing-logs",
+      "version": "1.0",
+      "handlers": [
+        {
+          "methods": ["GET"],
+          "pathPattern": "/metadata-provider/jobLogEntries/{jobExecutionId}/records/{recordId}",
```

Comment:
propose that we move this endpoint/handler to the previous `metadata-provider-job-execution-logs` interface, which will be responsible for providing data for logs lite